src/test/scala/com/fulcrumgenomics/fasta/SortSequenceDictionaryTest.scala

```diff
@@ -0,0 +1,230 @@
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.fulcrumgenomics.fasta
+
+import com.fulcrumgenomics.commons.io.PathUtil
+import com.fulcrumgenomics.testing.UnitSpec
+
+class SortSequenceDictionaryTest extends UnitSpec {
+  private val dir        = PathUtil.pathTo("src/test/resources/com/fulcrumgenomics/fasta")
+  private val chr1 = SequenceMetadata(
+    name             = "chr1",
+    length           = 100,
+    aliases          = Seq( "chr1_1", "chr1_2", "chr1_3"),
+  )
+
+  private val chr2 = SequenceMetadata(
+    name             = "chr2",
+    length           = 100,
+    aliases          = Seq( "chr2_1", "chr2_2", "chr2_3"),
+  )
+  private val chrM = SequenceMetadata(
+    name             = "chrM",
+    length           = 100,
+    aliases          = Seq( "chrM_1", "chrM_2", "chrM_3"),
+  )
+
+  private val chr1Alias = SequenceMetadata(
+    name="chr1_1",
+    length=100,
+  )
+  private val chr2Alias = SequenceMetadata(
+    name="chr2_1",
+    length=100,
+  )
+  private val chrMAlias = SequenceMetadata(
+    name="chrM_1",
+    length=100,
+  )
+
+  private val baseDictionary = SequenceDictionary(chr1, chr2, chrM)
+
+  "SortSequenceDictionary" should "Keep output identical to input if the sort order is the same" in {
+    val inputDictionaryPath  = makeTempFile("test.", ".1.dict")
+    val sortDictionaryPath = makeTempFile("test.", ".2.dict")
+    val outputDictionaryPath = makeTempFile("test.", ".3.dict")
+
+    baseDictionary.write(inputDictionaryPath)
+    baseDictionary.write(sortDictionaryPath)
+
+    val tool = new SortSequenceDictionary(
+      input         = inputDictionaryPath,
+      sortDictionary = sortDictionaryPath,
+      output        = outputDictionaryPath,
+    )
+    executeFgbioTool(tool)
+
+    val outputDictionary = SequenceDictionary(outputDictionaryPath)
+
+    outputDictionary.sameAs(baseDictionary) shouldBe true
+
+  }
+  
+  it should "Append metadata row if contig is present in input and not sortDictionary, and appendMissingContigs is True" in {
+    val inputDictionaryPath  = makeTempFile("test.", ".1.dict")
+    val sortDictionaryPath = makeTempFile("test.", ".2.dict")
+    val outputDictionaryPath = makeTempFile("test.", ".3.dict")
+
+    val subsetDictionary = SequenceDictionary(chr2, chrM)
+    val expectedDictionary = SequenceDictionary(chr2, chrM, chr1)
+
+    baseDictionary.write(inputDictionaryPath)
+    subsetDictionary.write(sortDictionaryPath)
+
+    val tool = new SortSequenceDictionary(
+      input         = inputDictionaryPath,
+      sortDictionary = sortDictionaryPath,
+      output        = outputDictionaryPath,
+    )
+    executeFgbioTool(tool)
+
+    val outputDictionary = SequenceDictionary(outputDictionaryPath)
+
+    outputDictionary.sameAs(expectedDictionary) shouldBe true
+  }
+
+  it should "Skip metadata row if contig is present in input and not sortDictionary, and appendMissingContigs is False" in {
```

Comment:
Change name since this makes it clearer to the user 
```suggestion
  it should "Skip metadata row if contig is present in input and not sortDictionary, and --append-missingContigs is False" in {
```